website/docs/docs/partnerships.mdx

```diff
@@ -187,8 +185,9 @@ For partners who are new to working with Hashicorp, we recommend [signing up for
 
 ### 2. Develop & Test
 
-Partners should build an integration using [Run Task APIs in Terraform Cloud](/cloud-docs/api-docs/run-tasks). To better understand how run Task enhances the workflow, see diagram listed below and check out our [announcement about Terraform run Task](https://www.hashicorp.com/blog/terraform-cloud-run-tasks-beta-now-available). [Snyk](https://docs.snyk.io/features/integrations/ci-cd-integrations/integrating-snyk-with-terraform-cloud), for example, created an integration to detect configuration anomalies in code while reducing risk to the infrastructure. For additional API resources, [click here](/cloud-docs/api-docs).
-**Currently, pre-apply is the only integration phase available.**
+Partners should build an integration using [Run task APIs in Terraform Cloud](/cloud-docs/api-docs/run-tasks). To better understand how post-plan run task enhances the workflow, see diagram listed below and check out our [announcement about Terraform run task](https://www.hashicorp.com/blog/terraform-cloud-run-tasks-beta-now-available). [Snyk](https://docs.snyk.io/features/integrations/ci-cd-integrations/integrating-snyk-with-terraform-cloud), for example, created an integration to detect configuration anomalies in code while reducing risk to the infrastructure. For additional API resources, [click here](/cloud-docs/api-docs).
```

Comment:
Do we need to link them to the additional API stuff if we already link to the Run Task APIs above?

```suggestion
Partners should build an integration using the [Run task APIs in Terraform Cloud](/cloud-docs/api-docs/run-tasks). For example, [Snyk](https://docs.snyk.io/features/integrations/ci-cd-integrations/integrating-snyk-with-terraform-cloud) created a run tasks to automatically detect misconfigurations in the Terraform plan. To learn more about how post-plan run tasks can enhance the Terraform Cloud workflow, refer to the following diagram and read our [run tasks announcement](https://www.hashicorp.com/blog/terraform-cloud-run-tasks-beta-now-available). 
```